apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -113,7 +114,7 @@ function SectionProgressSelector({
       ? showProgressTableV2
       : DCDO.get('progress-table-v2-default-v2', false));
 
-  const toggleV1OrV2Link = () => (
+  const ProgressV1OrV2ToggleLink = () => (
```

Comment:
Making this a component so I can configure it as one in the regional configuration.